### DISCUSSION TITLE
Rise of Horror Movies

### DISCUSSION THREAD

**Discussion by deepdk** [has2k1/plotnine#842] [2024 Plotnine Contest]
### Authors

Deepali Kank

### Links

[Twitter](https://twitter.com/DeepaliKank), [Github](https://github.com/deepdk), [LinkedIn](https://www.linkedin.com/in/deepali-kank/)

### Full description

The chart, represents the exponential growth of horror films released each year from 1950 to 2022. The visualization effectively illustrates both the growing popularity of horror movies and the financial success of key films over the last century.

![chart](https://github.com/user-attachments/assets/77df36fe-f38b-4811-abfa-77d8037b7180)

### Code repository

**Code**: https://github.com/deepdk/Plotnine_contest_2024

**Comment by NatashaData** [comment]
Hi Deepali,
Nice work! I like the colour scheme.
I think the this plot does effectively demonstrate the growing number of movies released, however I think the financial success could be better demonstrated - perhaps with a second axis?

**Comment by has2k1** [comment]
![chart](https://github.com/user-attachments/assets/49a03835-9847-48f7-ac88-92e3cd2f9757)

**Comment by EconomiCurtis** [comment]
Ahoy @deepdk , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@deepdk, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
